Kentor.AuthServices/SAML2P/Saml2PSecurityTokenHandler.cs

```diff
@@ -39,7 +39,7 @@ public Saml2PSecurityTokenHandler(EntityId spEntityId)
             {
                 IssuerNameRegistry = new ReturnRequestedIssuerNameRegistry(),
                 AudienceRestriction = audienceRestriction,
-                SaveBootstrapContext = FederatedAuthentication.FederationConfiguration.IdentityConfiguration.SaveBootstrapContext
+                SaveBootstrapContext = new FederationIdentityConfiguration().IdentityConfiguration.SaveBootstrapContext
```

Comment:
Not perfect, but I'm planning to ditch the Saml2PSecurityTokenHandler entirely soon, so let it be.